openwisp_radius/base/models.py

```diff
@@ -1007,7 +1007,12 @@ def get_or_create_user(self, row, users_list, password_length):
 
     def save_user(self, user):
         OrganizationUser = swapper.load_model('openwisp_users', 'OrganizationUser')
+        RegisteredUser = swapper.load_model('openwisp_radius', 'RegisteredUser')
         user.save()
+        registered_user = RegisteredUser(user=user)
+        if self.organization.radius_settings.needs_identity_verification:
+            registered_user.is_verified = True
```

Comment:
in the issue description I asked to use the "Manually created" option, see https://github.com/openwisp/openwisp-radius/blob/2235fe185bfb8e13118a3f9edc4e28126882ddc3/openwisp_radius/registration.py#L8-L13.